pdsc/ingest.py

```diff
@@ -172,11 +180,20 @@ def store_segments(outputfile, metadata, config):
     progress = standard_progress_bar('Segmenting footprints')
     for m in progress(metadata):
         try:
-            s = TriSegmentedFootprint(m, resolution, localizer_kwargs)
+            # erd: if not Mars, store body radius
+            if body_radius == MARS_RADIUS_M:
+                s = TriSegmentedFootprint(m, resolution, localizer_kwargs)
+            else:
+                s = TriSegmentedFootprint(m, resolution, localizer_kwargs, body_radius)
             for si in s.segments:
                 segments.append(si)
-                observation_ids.append(s.metadata.observation_id)
-
+                # erd: check if observation_id attribute exists
+                if hasattr(s.metadata, 'observation_id'):
+                    observation_ids.append(s.metadata.observation_id)
+                else:
+                    raise ValueError('Please rename your index col to observation_id in the config file')
+                    # erd: lroc does not have observation_id
+                    #observation_ids.append(s.metadata.file_specification_name)
```

Comment:
It might be preferable to check for this in the `store_metadata` function. We just want to check that some column is named `observation_id`. It should be a unique identifier, but does not need to be an indexed column.